src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
```

Comment:
The box-sizing property should also be set in the root element and inherited by other elements. It's a good practice to apply this property globally to ensure consistency across your styles.